process-automation/latest/model/forms.md

```diff
@@ -214,7 +214,7 @@ To create a conditional relationship between two dropdown fields using Country a
 
 **Note:** You can link as many dropdowns as you want.
 
-When you use dropdown menus in Process Automation you can select multiple entries in one dropdown menu to limit the amount of results in another checkbox.  
+When you use dropdown lists in Process Automation you can enter the first character of the item you are searching for to limit the amount of returned entries. This feature works when there are more than five entries and is useful when your lists are large. You can also select multiple entries in the dropdown list which is useful when you want to limit the returned entries of another dropdown list, that has a conditional relationship with it.
```

Comment:
You can search in a dropdown by any character(s), not only the first one. You can enter any part of the sentence you are looking for. I would change that in the description.

The second part is about "selection type", not searching. It's better to put it in a separate paragraph and describe to possible "selection types": single and multiple.

There is some inconsistency in the naming. At the top of section the dropdown is called "dropdown field" and here it's called "dropdown list". I'd rather use the same naming in all places.

Reply:
Resent for review, cheers Artur.